source/_UIAHandler.py

```diff
@@ -670,6 +677,34 @@ def IUIAutomationNotificationEventHandler_HandleNotificationEvent(
 			return
 		eventHandler.queueEvent("UIA_notification",obj, notificationKind=NotificationKind, notificationProcessing=NotificationProcessing, displayString=displayString, activityId=activityId)
 
+	def IUIAutomationActiveTextPositionChangedEventHandler_HandleActiveTextPositionChangedEvent(
+			self,
+			sender,
+			textRange
+	):
+		if not self.MTAThreadInitEvent.isSet():
+			# UIAHandler hasn't finished initialising yet, so just ignore this event.
+			if _isDebug():
+				log.debug("HandleActiveTextPositionchangedEvent: event received while not fully initialized")
+			return
+		import NVDAObjects.UIA
+		try:
+			obj = NVDAObjects.UIA.UIA(UIAElement=sender)
+		except Exception:
+			if _isDebug():

```

Comment:
I assume this would be a rare scenario? It might be worth having this log message show without specific debugging category enabled.

Reply:
All the other UIA event handlers also put this message behind _isDebug. In a perfect world this should not happen, but if the window is destroyed before we process the event, or perhaps the UIAElement dies before or as we make the NVDAObject, it could result in None. See ```source/NvDAObjects/__init__.py``` line 86.

Reply:
I think if it is a rare scenario, and useful debug information, it probably doesn't need to be hidden behind a category.